lib/utils/domFns.js

```diff
@@ -109,13 +109,23 @@ export function offsetXYFromParent(evt: {clientX: number, clientY: number}, offs
   return {x, y};
 }
 
-export function createCSSTransform({x, y}: {x: number, y: number}): Object {
+export function createCSSTransform({x, y}: {x: number, y: number}, defaultPosition: {x: number | string, y: number | string }): Object {
+  const defaultX = (typeof defaultPosition.x === 'string') ? defaultPosition.x : defaultPosition.x + 'px';
+  const defaultY = (typeof defaultPosition.y === 'string') ? defaultPosition.y : defaultPosition.y + 'px';
+  const translation = (defaultPosition && defaultPosition.x !== 0 && defaultPosition.y !== 0)
+  ? `translate(${defaultX}, ${defaultY}) translate(${x}px,${y}px)`
+  : 'translate(' + x + 'px,' + y + 'px)';
   // Replace unitless items with px
```

Comment:
This comment no longer makes sense with the changes above.

Reply:
Removed the comment